<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmaxExplorer</class>
 <widget class="QDialog" name="DmaxExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>305</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Dependent Variable:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dependentVariable">
           <item>
            <property name="text">
             <string>1-σ positive fraction</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>χ²/dof</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>I(Q=0)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Oscillation parameter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Background</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Positive Fraction</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Npts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Npts"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Min Distance [Å]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minDist"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Max Distance [Å]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxDist"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
